src/NServiceBus.Gateway/Settings/GatewaySettings.cs

```diff
@@ -36,5 +37,48 @@ public void ChannelFactories(Func<string, IChannelSender> senderFactory, Func<st
             config.GetSettings().Set("GatewayChannelSenderFactory", senderFactory);
             config.GetSettings().Set("GatewayChannelReceiverFactory", receiverFactory);
         }
+
+
+        /// <summary>
+        /// Set the number of retries and time increase between them for messages failing to be sent through the gateway.
+        /// </summary>
+        /// <param name="numberOfRetries">The total number of retries to do. 0 means no retry.</param>
+        /// <param name="timeIncrease">The time to wait between each retry.</param>
+        public void Retries(int numberOfRetries, TimeSpan timeIncrease)
```

Comment:
Talking to @SimonCropp and @timbussmann we propose that we go with `NServiceBus`